packages/material-ui-codemod/src/v4.0.0/top-level-imports.test/actual.js

```diff
@@ -54,7 +54,8 @@ import InputLabel from '@material-ui/core/InputLabel';
 import Input from '@material-ui/core/Input';
 import Grow from '@material-ui/core/Grow';
 import TableFooter from '@material-ui/core/TableFooter';
-import withWidth, { isWidthUp } from '@material-ui/core/withWidth';
+import withWidth from '@material-ui/core/withWidth';
+import { isWidthUp } from '@material-ui/core/withWidth';
```

Comment:
@oliviertassinari What happened here?

Reply:
Previously, `isWidthUp` wasn't exported from the barrel index.js. With the latest changes, imports can be changed. However, the codemod doesn't support the following case:
```js
import withWidth, { isWidthUp } from '@material-ui/core/withWidth';
```

It would need two runs to handle it correctly. This breaks the idempotent assertion.

I haven't found how to fix the 2 steps, instead, I have changed the import syntax to be idempotent.